### PR TITLE
Improve for project_spec

### PR DESCRIPTION
**Pull request by ydah**
This PR is improve for project_spec
- Fix the fact that entries was always an empty array and always succeeded because the format was changed by https://github.com/rubocop/rubocop-rspec/commit/4874a5a4a2a58e76d343aaa02279cd93b16f5a30 
- Add a test to check that there are no consecutive whitespace characters in the CHANGELOG
______________________________________________________________________

Before submitting the PR make sure the following are checked:

- [x] Feature branch is up-to-date with `master` (if not - rebase it).
- [x] Squashed related commits together.
- [x] Added tests.
- [x] Updated documentation.
- [-] Added an entry to the `CHANGELOG.md` if the new code introduces user-observable changes.
- [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

### CODE REVIEW

**Review by pirj — APPROVED** [review]
Thank you!
Does it make sense to add the commit to that file that would tell GitHub to ignore it for blame?

File: spec/project/changelog_spec.rb [hunk 1]
```diff
@@ -31,16 +31,16 @@
...
-      expect(entries).to all(match(/^\* \S/))
+      expect(entries).to all(match(/^- \S/))
     end
 
     it 'has a link to the contributors at the end' do
-      expect(entries).to all(match(/\(\[@\S+\]\[\](?:, \[@\S+\]\[\])*\)$/))
+      expect(entries).to all(match(/\(\[@\S+\](?:, \[@\S+\])*\)$/))
     end
 
     describe 'link to related issue on github' do
```
Comment (bquorning) [inline spec/project/changelog_spec.rb:46]: We have exactly _one_ such issue in the Changelog. Shouldn’t we just delete this example group?

File: spec/project/changelog_spec.rb [hunk 2]
```diff
@@ -31,16 +31,16 @@
   end
 
   describe 'entry' do
-    subject(:entries) { lines.grep(/^\*/).map(&:chomp) }
+    subject(:entries) { lines.grep(/^-/).map(&:chomp) }
 
     let(:lines) { changelog.each_line }
 
     it 'has a whitespace between the * and the body' do
```
[the remark this answers is not in the document]
Comment (ydah) [reply spec/project/changelog_spec.rb]: Indeed it is. Deleted.

File: spec/project/changelog_spec.rb [hunk 3]
```diff
@@ -31,16 +31,16 @@
   end
 
   describe 'entry' do
-    subject(:entries) { lines.grep(/^\*/).map(&:chomp) }
+    subject(:entries) { lines.grep(/^-/).map(&:chomp) }
```
[the remark this answers is not in the document]
Comment (ydah) [reply spec/project/changelog_spec.rb:34]: Yes, it is. We added test.

File: spec/project/changelog_spec.rb (hunk 1 above)
Comment (ydah) [reply spec/project/changelog_spec.rb]: You're right, we don't need that example group. I deleted it.

**Review by bquorning — APPROVED** [review]
Thank you 🙏🏼
